particles/PipeApps/source/RenderNotification.js

```diff
@@ -12,32 +12,29 @@
 
 defineParticle(({UiParticle, html, log}) => {
 
-  const notification = {
-    modality: 'notification',
-    title: `Hello`,
-    text: `I'm a notification.`,
-    handler: `onNotificationClick`
-  };
-
-  const clickNotification = {
-    modality: 'notification',
-    title: `Hello`,
-    text: `Notification was clicked`
-  };
-
   return class extends UiParticle {
-    renderOutput(inputs, state) {
-      if (!state.notified) {
-        state.notified = true;
-        this.output(notification);
-      }
-      if (state.clicked) {
-        state.clicked = false;
-        this.output(clickNotification);
-      }
+    setHandles(handles) {
+      this.output({
```

Comment:
please, sync this file. 
the particle now overrides `renderOutput` (instead of `setHandles`).
you should probably add another constant:
```
  const dismissNotification = {
    modality: 'notification',
    title: `Hello`,
    text: `Notification was dismissed`
  };
```
and a new `dismissed` boolean in the `state`

Reply:
Done.